src/webhook/subscription_validation.py

```diff
@@ -1,18 +1,26 @@
-from flask import request, Blueprint
+from flask import request, Blueprint, Response
 import sys, traceback
 import json
 import os
+import prometheus_client
+from prometheus_client import Counter
+from prometheus_client.core import CollectorRegistry
 
 from webhook.request_helper import validate, responses
 
 bp = Blueprint("subscription-webhook", __name__)
 
-valid_source_namespaces = os.getenv("SUBSCRIPTION_VALIDATION_NAMESPACES", "openshift-marketplace")
+# define what we track, declare Counter, how many times this route is accessed
+TOTAL_SUBSCRIPTION = Counter('webhook_subscription_validation', 'The total number of subscription validation requests')
```

Comment:
Same naming standard from the other webhook applies here.

Reply:
fixed